ui/app/pages/error/error.component.js

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+class ErrorPage extends Component {
+  renderErrorDetail (content) {
+    return (
+      <li>
+        <p>
+          {content}
+        </p>
+      </li>
+    )
+  }
+
+  renderErrorStack (title, stack) {
+    return (
+      <li>
+        <span>
+          {title}
+        </span>
+        <pre className="error-page__stack">
+          {stack}
+        </pre>
+      </li>
+    )
+  }
+
+  render () {
+    const { error } = this.props
+    const { t } = this.context
+
+    return (
+      <section className="error-page">
+        <h1 className="error-page__header">
+          {t('errorPageTitle')}
+        </h1>
+        <h2 className="error-page__subheader">
+          {t('errorPageMessage')}
+        </h2>
+        <section className="error-page__details">
+          <details>
+            <summary>
+              {t('errorDetails')}
+            </summary>
+            <ul>
+              { error.message ? this.renderErrorDetail(t('errorMessage', [error.message])) : null }
+              { error.code ? this.renderErrorDetail(t('errorCode', [error.code])) : null }
+              { error.name ? this.renderErrorDetail(t('errorName', [error.name])) : null }
+              { error.stack ? this.renderErrorStack(t('errorStack'), error.stack) : null }
+            </ul>
+          </details>
+        </section>
+      </section>
+    )
+  }
+}
+
+ErrorPage.contextTypes = {
```

Comment:
Yes - I will update this. This is leftover from an earlier version where `ErrorPage` was a functional component. I had to make it a class to get access to `context`. There is a way to do it with `useContext` instead, but I couldn't sue that because we're not using the newer context API 😞